explore/src/main/scala/explore/components/ColumnSelectorInTitle.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import cats.syntax.all.*
+import crystal.react.View
+import explore.components.ui.ExploreStyles
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.react.common.ReactFnProps
+import lucuma.react.table.ColumnId
+import lucuma.react.table.Table
+import lucuma.ui.table.ColumnSelector
+
+case class ColumnSelectorState[A, B](
+  table: Option[Table[A, B]] = None
+)
+
+case class ColumnSelectorInTitle[A, B](
```

Comment:
This was extracted to be common across several tiles

Reply:
Nice